src/engine/engine.rs

```diff
@@ -109,7 +137,7 @@ pub trait Engine: Debug {
                    blockdev_paths: &[&Path],
                    redundancy: Option<u16>,
                    force: bool)
-                   -> EngineResult<(PoolUuid, Vec<PathBuf>)>;
+                   -> EngineResult<PoolUuid>;
```

Comment:
This change seems good...header comment needs updating.

Reply:
ok